### PR TITLE
Uninstall resolvconf + jinja rendering bug fix

### PR DESCRIPTION
**Pull request by ryepup**
On debian, ensures the `resolvconf` package is not installed. If it's installed, then it will overwrite resolv.conf. Also fixes a rendering bug where the `search` option was being rendered on the same line as a comment.

### CODE REVIEW

File: resolver/init.sls [hunk 1]
```diff
@@ -2,7 +2,13 @@
 ##### Salt Formula For Resolver #####
 #####################################
 
+{% from "resolver/map.jinja" import resolver with context %}
+
 # Resolver Configuration
+resolver:
+  pkg.removed:
+    - name: {{resolver.package}}
```
Comment (babilen) [inline resolver/init.sls]: You might want to adjust whitespace here to `{{ resolver.package }}` as in the other cases.

Comment (ryepup) [inline resolver/init.sls]: fixed

**Comment by 0xf10e** [comment]
I recomment configuring `resolvconf` instead of removing the package.
I'm rather sure it will be reinstalled during a release-upgrade on Ubuntu and the same might happen on Debian thus you might end up with a broken `resolv.conf`.

Just place the comment in `/etc/resolvconf/resolv.conf.d/head` and the rest in `/etc/resolvconf/resolv.conf.d/base`.

PS: I've done this in the [OVS-formula](https://github.com/saltstack-formulas/openvswitch-formula/blob/master/networking/resolvconf.sls).

**Comment by babilen** [comment]
I am with @0xf10e on this one.

**Comment by aboe76** [comment]
@ryepup and @0xf10e can this issue be resolved? should we use resolvconf/resolv.conf.d/head?

if so @0xf10e can you create a PR?

**Pull request closed without merging**